envoystat/README.md

```diff
@@ -0,0 +1,84 @@
+# envoystat
+
+envoystat displays information from the envoy stats endpoint at an interval. This is useful for
+monitoring the behavior of a system in real-time.
+
+```
+$ python envoystat.py 
+2018/04/04 envoy 19745a5e3d267b8621f6b0740250bbe04af3fadc/1.7.0-dev/Clean/RELEASE live 66476 66476 0
+
+03:57:30 PM cx_active- rq_active-     rq_2xx     rq_4xx     rq_5xx
+03:57:31 PM        371          3         94          0          4
+03:57:32 PM        371          5         66          3          5
+03:57:33 PM        371          1         99          0          6
+03:57:34 PM        371          0         58          9          2
+03:57:35 PM        371          2         51          0          1
+^C
+```
+
+The `-` suffix denotes a gauge. All other values are assumed to be counters. At each interval, the
+script will display the difference for each counter and the current value of each gauge passed in
+the `--fields` parameter.
+
+# Dependencies
+
+envoystat has no dependencies other than Python 2.
+
+# Installation
+
+Run directly with `python envoystat.py` or symlink it into the system path. In the future, envoystat
+will be a Python module installable via pip.
+  
+# Usage
+
+`envoystat [--admin <uri>] [--prefix <prefix>] [--fields <field ...>] [--interval <interval>]`
+
+`-a, --admin`
+> Admin URI. The script will request the `/stats` endpoint from this URI. Defaults to `http://localhost:9901`.
```

Comment:
Nit: where does 9901 come from? Is this in Envoy example configs?

Reply:
That is our admin port at Lyft. I didn't see a consistent port in the examples.